app/src/main/java/com/locoquest/app/RetrofitClientInstance.kt

```diff
@@ -2,10 +2,11 @@ package com.locoquest.app
 
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.converter.moshi.MoshiConverterFactory
 
-class RetrofitClientInstance {
+object RetrofitClientInstance {
```

Comment:
RetrofitClientInstance should be an object as it is a static class and we use a static class when it's just a tool that works on other "stuff".
https://stackoverflow.com/questions/1185605/when-to-use-static-vs-instantiated-classes

Reply:
Good suggestion, though this same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.